vcell-core/src/main/java/org/vcell/solver/langevin/LangevinSolver.java

```diff
@@ -197,9 +197,13 @@ protected String[] getMathExecutableCommand() {
 		String messagingConfigOption = "--vc-send-status-config=" + getMessagingConfigFilename();
 		String localMessagingOption = "--vc-print-status";
 		
-		// TODO: we'll probably eliminate this as command line argument, or set it correctly at some later time
-		// TODO: the correct value is the SolverTaskDescription TrialIndex variable
-		int trialIndex = 0;		// run index
+		int trialIndex;		// run index
```

Comment:
int trialIndex = simTask.getSimulationJob().getTrialIndex().index

Reply:
sorry for being a stickler ... there is no exception thrown (I'm pretty sure), and keeping the try/catch is confusing.

please remove the try/catch, and just keep the assignment.